src/ensembl/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow: /
+Host: 2020.ensembl.org
```

Comment:
Ok, I see that we already need the programmatic control over the generation of robots.txt.

The robots.txt on the 2020 site should say that the host is `2020.ensembl.org`, while the robots.txt on the staging-2020 site should say that the host is `staging-2020.ensembl.org`.